elide-model-config/src/main/java/com/yahoo/elide/modelconfig/parser/handlebars/HandlebarsHelper.java

```diff
@@ -24,7 +25,7 @@
  */
 public class HandlebarsHelper {
 
-    private static final String EMPTY_STRING = "";
+    public static final String EMPTY_STRING = "";
```

Comment:
It looks like this is not accessed outside this class.

Reply:
used in DynamicConfigValidator.

Reply:
reverted now.